manual/commands.md

```diff
@@ -0,0 +1,633 @@
+# Help
+Once installed, if you run the `meza` command all by itself, it will display *help* like the following:
+```bash
+Mediawiki EZ Admin
+
+Usage: meza COMMAND [directives]
+
+To setup a multi-server environment, do:
+
+$ meza setup env # Setup the environment, following prompts
```

Comment:
Does this need to be done from a particular directory? Can it be done as a regular user?